fe/fe-core/src/main/java/com/starrocks/load/loadv2/BulkLoadJob.java

```diff
@@ -252,40 +250,10 @@ public void onTaskFailed(long taskId, FailMsg failMsg) {
                         .build());
                 return;
             }
-            LoadTask loadTask = idToTasks.get(taskId);
-            if (loadTask == null) {
-                return;
-            }
-            if (loadTask.getRetryTime() <= 0) {
+
+            if (!failMsg.getMsg().contains("timeout")) {
```

Comment:
better use cancelType to check

Reply:
The cancelType is not accurate now. The timeout most cancelType is CANCELLED, not TIMEOUT.